plugins/azure-devops/README.md

```diff
@@ -313,3 +313,62 @@ To get the README component working you'll need to do the following two steps:
 - You'll need to add the `EntitySwitch.Case` above from step 2 to all the entity sections you want to see Readme in. For example if you wanted to see Readme when looking at Website entities then you would need to add this to the `websiteEntityPage` section.
 - The `if` prop is optional on the `EntitySwitch.Case`, you can remove it if you always want to see the tab even if the entity being viewed does not have the needed annotation
 - The `maxHeight` property on the `EntityAzureReadmeCard` will set the maximum screen size you would like to see, if not set it will default to 100%
+
+## Permission Framework
+
+Azure DevOps plugin is now supporting the permission framework for PRs, GitTags, Pipelines & ReadMe.
+
+To enable the permission on the legacy backend system add the below config in `packages/backend/src/plugins/azure-devops.ts`.
+
+```diff
+export default async function createPlugin(
+  env: PluginEnvironment,
+): Promise<Router> {
+  return createRouter({
+    logger: env.logger,
+    config: env.config,
+    reader: env.reader,
++   permissions: env.permissions,
+  });
+}
+```
+
+### Configure Permission
+
+To apply the permission rule add the following in `packages/backend/src/plugins/permissions.ts`.
+
+> Note this is example only `azureDevOpsPullRequestDashboardReadPermission` is Basic permission rest all Resource Permission. As an adopter you can configure how you wanted.
+
+```diff
++ import {
++  azureDevOpsPullRequestReadPermission,
++  azureDevOpsPipelineReadPermission,
++  azureDevOpsGitTagReadPermission,
++  azureDevOpsReadmeReadPermission,
++  azureDevOpsPullRequestDashboardReadPermission } from '@backstage/azure-devops-common';
```

Comment:
```suggestion
+  azureDevOpsPullRequestDashboardReadPermission } from '@backstage/plugin-azure-devops-common';
```

Reply:
There are more imports needed to make this work. I need to add:

`isPermission` from `@backstage/plugin-permission-common`

also:

```
import {
  catalogConditions,
  createCatalogConditionalDecision,
} from '@backstage/plugin-catalog-backend/alpha';
```

Reply:
Instead of adding the other imports, will add like below on the top the `Permissions` section. WDYT?

> This has the prerequisite to integrate Permission Framework [Getting-started](https://backstage.io/docs/permissions/getting-started)

Reply:
I'd prefer to have as close to a working example as possible, what I found is most people are happy with the defaults we provide in these examples. This also helps reduce the number of questions that will come up about this either in Issue or on Discord. 👍